h2/src/test/org/h2/test/db/TestCompatibility.java

```diff
@@ -414,6 +414,7 @@ private void testMySQL() throws SQLException {
         stat.execute("CREATE TABLE TEST_12(ID INT) DEFAULT COLLATE UTF8");
         stat.execute("CREATE TABLE TEST_13(a VARCHAR(10) COLLATE UTF8MB4)");
         stat.execute("CREATE TABLE TEST_14(a VARCHAR(10) NULL CHARACTER SET UTF8MB4 COLLATE UTF8MB4_BIN)");
+        stat.execute("ALTER TABLE TEST_14 CONVERT TO CHARACTER SET UTF8MB4 COLLATE UTF8");
```

Comment:
Maybe use some collation supported by MySQL and / or MariaDB? Both these DBMS say that collation `UTF8` isn't known.

Reply:
I hadn't tested collation, thanks for warning for the fix :)